{
  "id": "ALSA-2024:5530",
  "summary": "Important: python-setuptools security update",
  "affected": [
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "platform-python-setuptools"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "39.2.0-8.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3-setuptools"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "39.2.0-8.el8_10"
            }
          ]
        }
      ]
    },
    {
      "package": {
        "ecosystem": "AlmaLinux:8",
        "name": "python3-setuptools-wheel"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "39.2.0-8.el8_10"
            }
          ]
        }
      ]
    }
  ],
  "related": [
    "CVE-2024-6345"
  ],
  "published": "2024-08-19T00:00:00Z",
  "modified": "2024-08-21T12:35:03Z",
  "details": "The python-setuptools package provides a collection of enhancements to Python distribution utilities allowing convenient building and distribution of Python packages.\n\nSecurity Fix(es):\n\n* pypa/setuptools: Remote code execution via download functions in the package_index module in pypa/setuptools (CVE-2024-6345)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.",
  "references": [
    {
      "url": "https://access.redhat.com/errata/RHSA-2024:5530",
      "type": "ADVISORY"
    },
    {
      "url": "https://access.redhat.com/security/cve/CVE-2024-6345",
      "type": "REPORT"
    },
    {
      "url": "https://bugzilla.redhat.com/2297771",
      "type": "REPORT"
    },
    {
      "url": "https://errata.almalinux.org/8/ALSA-2024-5530.html",
      "type": "ADVISORY"
    }
  ]
}